src/test/project-manager-helpers.ts

```diff
@@ -0,0 +1,35 @@
+import * as chai from 'chai';
+import { ProjectManager } from '../project-manager';
+import chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+const assert = chai.assert;
+
+export type ProjectManagerFactory = () => ProjectManager;
+
+export const expectedFiles: string[][] = [
+	['package.json', '{"name": "package-name-1"}'],
+	['subdirectory-with-tsconfig/package.json', '{"name": "package-name-2"}'],
+	['subdirectory-with-tsconfig/src/tsconfig.json', '{}'],
+	['subdirectory-with-tsconfig/src/dummy.ts', '']
+];
+
+/**
+ * Describe a ProjectManager class
+ *
+ * @param createService Factory function to create the TypeScriptService instance to describe
+ */
+export async function describeProjectManager(factory: ProjectManagerFactory) {
```

Comment:
I don't think we need this, we only need to support subclassing `TypeScriptService`